File: tool/templates/template.nim
from times import cpuTime
from os import paramStr

proc run(s: string): string =
    # Your code here
    return ""


var input: string = paramStr(1)

var t0 = cpuTime()
var output = run(input)

echo "_duration:", (cpuTime() - t0) * 1000
echo output

File: tool/templates/template.py
from tool.runners.python import SubmissionPy


class {class_name}(SubmissionPy):
    def run(self, s):
        """
        :param s: input in string format
        :return: solution flag
        """
        # Your code goes here
        pass


def test_{author_name}():
    """
    Run `python -m pytest {submission_file}` to test the submission.
    """
    assert (
        {class_name}().run(
            """
""".strip()
        )
        == None
    )
